src/components/NewTodo/NewTodo.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from 'react';
+import users from '../../api/users';
+
+interface Props {
+  onAdd: (todo: TodoCreateData) => void;
+}
+
+export type TodoCreateData = {
+  title: string;
+  userId: number;
+};
+
+export const NewTodo: React.FC<Props> = ({ onAdd }) => {
+  const [title, setTitle] = useState('');
+  const [titleError, setTitleError] = useState('');
+  const [userId, setUserId] = useState(0);
+  const [userIdError, setUserIdError] = useState('');
+
+  return (
+    <form
+      action="/api/todos"
+      method="POST"
+      onSubmit={event => {
+        event.preventDefault();
+        if (!title) {
+          setTitleError('Please enter a title');
+        }
+
+        if (!userId) {
+          setUserIdError('Please choose a user');
+        }
+
+        if (title && userId) {
+          onAdd({ title, userId });
+          setTitle('');
+          setUserId(0);
+        }
+      }}
+    >
+      <div className="field">
+        <label>
+          Title:
+          <input
+            type="text"
+            value={title}
+            data-cy="titleInput"
+            placeholder="Please enter a title"
+            onChange={event => {
+              setTitle(event.target.value);
+              setTitleError('');
+            }}
```

Comment:
Create handlers
Don't pass anonim functions to onChange/etc if they have more the one action to do, create handlers instead